doc/browser-capabilities.md

```diff
@@ -0,0 +1,44 @@
+# Jitsi Meet browser capabilities
+
+An API for checking browser capabilities for Jitsi Meet.
+
+## Installation
+
+To use the Jitsi Meet browser capabilities API in your application you need to add a JS file with `BrowserCapabilities` implementation:
+
+```javascript
+<script src="https://meet.jit.si/libs/browser_capabilities.min.js"></script>
```

Comment:
Perhaps change that to example.com to make it clear that the file is specific to the deployment